docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '2'
+services:
+    node:
```

Comment:
Small suggestion, shall we use name `api` instead `node`?

Reply:
Changed it as well in the new commit